File: module_mine/src/main/java/com/xiaoyv/mine/main/contract/MineContract.java
package com.xiaoyv.mine.main.contract;

/**
 * MineContract
 *
 * @author why
 * @since 2020/12/10
 **/
public interface MineContract {
    interface Adapter {
        /**
         * 调换位置
         *
         * @param fromPos 从
         * @param toPos   至
         */
        void onItemChange(int fromPos, int toPos);

        /**
         * 删除条目
         *
         * @param pos 位置
         */
        void onItemDelete(int pos);
    }
}

File: module_mine/src/main/java/com/xiaoyv/mine/main/contract/MineContract.kt
package com.xiaoyv.mine.main.contract

/**
 * MineContract
 *
 * @author why
 * @since 2020/12/10
 */
interface MineContract {

    interface Adapter {
        /**
         * 调换位置
         *
         * @param fromPos 从
         * @param toPos   至
         */
        fun onItemChange(fromPos: Int, toPos: Int)

        /**
         * 删除条目
         *
         * @param pos 位置
         */
        fun onItemDelete(pos: Int)
    }
}